middlewares/logger-middleware.ts

```diff
@@ -0,0 +1,53 @@
+import {PolarisMiddleware} from "./polaris-middleware";
+import {inject, injectable} from "inversify";
+import {PolarisLogger} from "@enigmatis/polaris-logs/dist/src/polaris-logger";
+import {InjectableLogger} from "..";
+import {GraphQLResolveInfo} from "graphql";
+import {GraphQLLogProperties} from "../logging/GraphQLLogProperties";
+import {RequestLogProperty} from "@enigmatis/polaris-logs/dist/src/entities/request-log-property";
+import {PolarisLogProperties} from "@enigmatis/polaris-logs/dist/src/polaris-log-properties";
+
+const graphqlFields = require('graphql-fields');
+
+@injectable()
+export class LoggerMiddleware implements PolarisMiddleware {
+    @inject("InjectableLogger") polarisLogger: InjectableLogger;
+
+    initReqProperties(root, args, context, info: GraphQLResolveInfo): GraphQLLogProperties {
+        let fieldsWithSubFieldsArgs = graphqlFields(info, {}, {processArguments: true});
+        let body = {}
+        body[info.fieldName] = fieldsWithSubFieldsArgs;
+        let req: RequestLogProperty = {
+            requestQuery: {
+                body: JSON.stringify(body).replace(/:{}/g, "")
+            }
+        };
+        let prop: GraphQLLogProperties = {
+            operationName: info.operation.name.value,
+            request: req
+        };
+        return prop;
+    }
+
+    preResolve(root:any, args:{ [argName: string]: any }, context:any, info:GraphQLResolveInfo) {
+        if (!root) {
+            let props: GraphQLLogProperties = this.initReqProperties(root, args, context, info);
+            this.polarisLogger.debug('Resolver of ' + props.operationName +
```

Comment:
Messages to const, and look at the following example to format a string:

var a = "Hello";
var b = "World";

var text = \`${a} ${b}\`